test/compound/TestSupplyVault.t.sol

```diff
@@ -469,21 +469,6 @@ contract TestSupplyVault is TestSetupVaults {
         assertApproxEqAbs(uint256(userReward1_1), userReward1_2, 1);
     }
 
-    // TODO: fix this test by using updated indexes in previewMint
-    // function testShouldMintCorrectAmountWhenMorphoPoolIndexesOutdated() public {
```

Comment:
Did you re-try this test? Perhaps redundant to new tests though..

Reply:
This test is redundant yes

Reply:
And it is passing, except on aave-v3 on which the lens was not added